mem.go

```diff
@@ -2,12 +2,60 @@ package sqrl
 
 import (
 	"fmt"
+	"math/big"
+	"os"
 
+	"github.com/dustin/go-humanize"
+	"github.com/likexian/host-stat-go"
 	"github.com/shirou/gopsutil/mem"
 )
 
-func MemInfo() {
-	v, _ := mem.VirtualMemory()
+type Ram struct {
+	total, free, swapSpace string
+}
+
+type OS struct {
+	hostname, kernel, version string
+}
+
+func ramOSInfo() (Ram, OS) {
+	v, e := mem.VirtualMemory()
+	if e != nil {
+		fmt.Println(e)
```

Comment:
This is much better, but still poses an issue. 

So lets say I call this function.  How would the caller of this function know that something happened? Just on stdout, which wouldn't serve them very well inside of the program.  What you should probably do here is something more along the lines of:

``` go
if err != nil {
    return Ram{}, OS{}, err
}
```

and then the caller of this function can know that something went wrong when calling it.  Does that make sense?